<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestingDialog</class>
 <widget class="QDialog" name="TestingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>25</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLCDNumber" name="time_Number">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>0</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>103</width>
     <height>180</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Filled</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableView" name="tbview_TestResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>461</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QHeaderView {
    color:rgb(255, 255, 255);
    font: bold 26pt;
    background-color: rgb(85, 170, 255);
    border: 0px solid rgb(144, 144, 144);
    border:0px solid rgb(191,191,191);
    border-left-color: rgba(255, 255, 255, 0);
    border-top-color: rgba(255, 255, 255, 0);
    border-radius:0px;
    min-height:29px;
	 selection-background-color: rgb(77, 77, 77);  
    selection-color: white;  
}
 
QTableView{
color:rgb(0, 85, 255);
}

QScrollBar:horizontal{height:25px;background:transparent;background-color:rgb(249, 249, 249);margin:0px,0px,0px,0px;padding-left:25px;padding-right:25px;}

QScrollBar::handle:horizontal{height:25px;background:lightgray;border-radius:6px;}

QScrollBar::handle:horizontal:hover{height:25px;background:rgb(0, 85, 127);border-radius:6px;}



QScrollBar:vertical{width:25px;background:transparent;background-color:rgb(249, 249, 249);margin:0px,0px,0px,0px;padding-top:25px;padding-bottom:25px;}
QScrollBar::handle:vertical{width:25px;background:lightgray ;border-radius:5px;min-height:20px;}
QScrollBar::handle:vertical:hover{width:25px;background:rgb(0, 85, 127);border-radius:5px;min-height:20px;}</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::AllEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_test">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>-6</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Serif</family>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 170, 255); </string>
   </property>
   <property name="text">
    <string>Status:Strip Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Start">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>643</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: white;
border-radius: 20px;
border: 2px groove gray;
border-style: outset;
}
QPushButton:hover{
background-color:white;
color: black;
}
QPushButton:pressed{
background-color:rgb(0, 0, 0);
color: white;
border-style: inset;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/image/Run.png</normaloff>:/image/Run.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Back">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>643</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: white;
border-radius: 20px;
border: 2px groove gray;
border-style: outset;
}
QPushButton:hover{
background-color:white;
color: black;
}
QPushButton:pressed{
background-color:rgb(0, 0, 0);
color: white;
border-style: inset;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/image/Home.png</normaloff>:/image/Home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Stop">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>643</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: white;
border-radius: 20px;
border: 2px groove gray;
border-style: outset;
}
QPushButton:hover{
background-color:white;
color: black;
}
QPushButton:pressed{
background-color:rgb(0, 0, 0);
color: white;
border-style: inset;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/image/Stop.png</normaloff>:/image/Stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Start">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>706</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Serif</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Stop">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>706</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Serif</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Back">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>706</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Serif</family>
     <pointsize>29</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 170, 255);</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_process">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>6</y>
     <width>82</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_Start</tabstop>
  <tabstop>btn_Stop</tabstop>
  <tabstop>btn_Back</tabstop>
 </tabstops>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
